node-server/.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules
 #coverage folder
 coverage
 
-.idea
\ No newline at end of file
+.idea
+.vscode
```

Comment:
For the record I recommend everybody add their personal IDE specific files to a `.gitignore_global` file in their home directory.  But I don't have a problem adding this here as well.